fastlane/spec/actions_specs/git_commit_spec.rb

```diff
@@ -3,6 +3,7 @@
     describe "git_commit" do
       before :each do
         allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return(nil)
+        allow(FastlaneCore::FastlaneFolder).to receive(:skip_git_hooks).and_return(nil)
```

Comment:
You can remove this line ^ `skip_git_hooks` doesn't get called on `FastlaneFolder` at all 👌